tests/phpunit/tests/theme/globalStylesheet.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/**
+ * @group themes
+ */
+class Tests_Global_Stylesheet extends WP_UnitTestCase {
```

Comment:
The name of the class should be Tests_Theme_GlobalStylesheet following the pattern for the other tests.

Reply:
https://github.com/WordPress/wordpress-develop/pull/2233/commits/160070e05281de3eb9848f6e4b6eb500757c674b